packages/popover/src/utils/getPopoverRenderModeProps.ts

```diff
@@ -0,0 +1,43 @@
+import { PopoverRenderModeProps, RenderMode } from '../Popover';
+
+/**
+ * Util function that returns relevant properties based on the `renderMode` prop
+ * @internal
+ */
+export const getPopoverRenderModeProps = ({
+  dismissMode,
+  onToggle,
+  portalClassName,
+  portalContainer,
+  portalRef,
+  renderMode,
+  scrollContainer,
```

Comment:
nit, would it make sense to pull out the arg type? Don't think we have guidelines, but imo if the arg object has more than 2-3 properties, I prefer to extract the object into something like `interface GetPopoverRenderModePropsArgs` (preferably less verbose...)